src/NJsonSchema/NJsonSchema.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard1.0;netstandard2.0;net40;net45</TargetFrameworks>
     <Description>JSON Schema reader, generator and validator for .NET</Description>
-    <Version>10.0.21</Version>
+    <Version>20.0.21</Version>
```

Comment:
Please do not change version

Reply:
It was an accident. I've already corrected.